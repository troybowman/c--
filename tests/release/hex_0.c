../real/hex_0.c