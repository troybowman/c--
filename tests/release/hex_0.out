0 10 22 305419896 43981
26796
41506
38163215
-4369
-8738
